### PR TITLE
JavaDoc addition

### PR DESCRIPTION
**Pull request by JakobBrownie**
I have not seen too much Java Documentation, so I added some here. I am not seeing many issues and this seemed like one that nobody else has covered.

### CODE REVIEW

**Comment by JakobBrownie** [comment]
Analysis of the Program:

The program was available on time.
The program was easy to read and understand.
The program does compile.

https://github.com/bolandmj/TourGuide/tree/Brown-Jakob-database-1
I only have one change and it is not merged yet because I created and populated our database for our project.
This was the task that I was given for our sprint so I did not have the opportunity to commit more.

I learned how the @springbootapplication works.
I learned how the @springboottest works.
I learned how the MockMvc works in application.

**
package com.laborhelper.enterprise;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class LaborHelperApplication {

    public static void main(String[] args) {
        SpringApplication.run(LaborHelperApplication.class, args);
    }

}

**
@SpringBootTest
class LaborHelperApplicationTests {

    @Test
    void contextLoads() {
    }

}
@WebMvcTest
class WebLayerTest {

    @Autowired
    private MockMvc mockMvc;

    @Test
    public void shouldReturnDefaultMessage() throws Exception {
        this.mockMvc.perform(get("/")).andDo(print()).andExpect(status().isOk())
                .andExpect(content().string(containsString("Hello World")));

File: src/main/java/com/laborhelper/enterprise/LaborHelperApplication.java [hunk 1]
```diff
@@ -1,4 +1,7 @@
 package com.laborhelper.enterprise;
+/**
+*The use of @SpringBootApplication runs everything that @configuration, @EnableAutoConfiguration, and @ComponentScan with the default attributes
+*/
```
Comment (discospiff) [inline src/main/java/com/laborhelper/enterprise/LaborHelperApplication.java:4]: 1) This belongs above the class.
2) Most of this is a given to those who understand Spring Boot... so the comment isn't adding much value.

File: src/main/java/com/laborhelper/enterprise/dto/LaborAction.java [hunk 2]
```diff
@@ -5,6 +5,10 @@
 import java.util.Date;
 import java.util.Map;
 
+/**
+*@NonNull lombok generates a null-check statement
```
Comment (discospiff) [inline src/main/java/com/laborhelper/enterprise/dto/LaborAction.java:9]: This is describing Lombok, not the project itself.

**Comment by discospiff** [comment]
JavaDoc is a good idea, but it should add value over and above what can be determined by context.
No need to merge this branch as it currently is.

**Pull request closed without merging**